marko.json

```diff
@@ -66,7 +66,10 @@
         "@heading-level": "string",
         "@status": "string",
         "@class": "string",
+        "@hidden": "boolean",
         "@aria-text": "string",
+        "@aria-close": "string",
```

Comment:
I'm not a fan of `aria-close`, as `aria` is a bit ambiguous (i.e. aria is not just used for labels). Maybe `close-icon-label` or `aria-label-close` instead.

Let's try and be consistent across the components and all decide on a convention. We can use slack if necessary.

In hindsight `aria-text` is a bit ambiguous too (but we all reviewed the PR, so not your fault!). I might rename that at some point.

Reply:
I thought `aria-label-close` would become a too long name so I adopted for aria-close but I can change it as its more descriptive